.github/workflows/release-please.yml

```
@@ -21,9 +26,25 @@ concurrency:
+
+  build-assets:
+    # Fire only when release-please actually cut a new release on this run.
+    # `release_created` is the v4 output flag; `tag_name` carries the cut tag.
+    needs: release-please
+    if: ${{ needs.release-please.outputs.release_created == 'true' }}
+    uses: ./.github/workflows/rebuild-release.yml
+    permissions:
+      contents: write
+    with:
+      tag: ${{ needs.release-please.outputs.tag_name }}
+      force: "true"
```

actionlint (1.7.12)

[error] input "force" is typed as string by reusable workflow "./.github/workflows/rebuild-release.yml". bool value cannot be assigned

(expression)